tensorboard/defs/defs.bzl

```diff
@@ -149,3 +149,29 @@ def tf_ng_module(assets = [], **kwargs):
         angular_assets = assets,
         **kwargs
     )
+
+def tf_res_digest_suffixer(name, resources, template, out):
```

Comment:
Can we use the full word `resource` in this name?

I’m not in love with `tf_resource_digest_suffixer`, because most rules
are named based on what the output is (a binary, a library, a test)
rather than a “«verb»er”. But I don’t immediately have a suggestion, so
no action required.